### ISSUE TITLE
[BUG] Progress bar crashes at the end of a song

### ISSUE THREAD

**Issue by 4Lajf** [SushiBtw/discord-music-player#84]
**Bug Description**
If you use example code for progress bar and call it at the end of current song when there is no songs in queue the bot will crash.
**Reproduction Steps**
1. Use example command for progress bar
2. Call it at the end of current song when there is nothing more in queue

**Expected Result**
Error message: `There is no music playing right now.`

**Versioning:**
Node v15.12.0
Npm 7.6.3
discord-music-player@7.0.0-fix3

**Additional Comments**
```
        message.channel.send(`**${newSong.name}** is now playing!`))
                                          ^

TypeError: Cannot read property 'name' of undefined
    at Player.<anonymous> (C:\Users\SzK\Documents\DBM\Karren\music.js:37:43)
    at Player.emit (node:events:369:20)
    at Player._playSong (C:\Users\SzK\Documents\DBM\Karren\node_modules\discord-music-player\src\Player.js:810:18)
    at StreamDispatcher.<anonymous> (C:\Users\SzK\Documents\DBM\Karren\node_modules\discord-music-player\src\Player.js:859:39)
    at StreamDispatcher.emit (node:events:381:22)
    at finish (node:internal/streams/writable:741:10)
    at processTicksAndRejections (node:internal/process/task_queues:81:21)
```

**Comment by SushiBtw** [comment]
I will be looking into those crash reports!

**Comment by SushiBtw** [comment]
I am working on a `v7.0.1` update dropping today! Fix incoming.

**Comment by SushiBtw** [comment]
Hey! Update time!
I hope everything got fixed, please check within `v7.0.1` and let me know!
https://www.npmjs.com/package/discord-music-player

Thank you for the patience,
Sushi!
